### PR TITLE
Update de-DE.json

### PR DESCRIPTION
**Pull request by MickLesk**
Update German Language
-> Fix missing translations
-> Strange translations fixed / optimized

### CODE REVIEW

**Review by danieldietzler — CHANGES_REQUESTED** [review]
You shouldn't change that here but instead ask @alextran1502 so that he can invite you to the localization platform. Changing this file causes synchronization issues

**Comment by alextran1502** [comment]
@danieldietzler Since he made a lot of changes already, I can update localizely with this file

**Comment by MickLesk** [comment]
I didn't know that :-) I thought I'd fix a few little things :P 

But you are welcome to invite me to the localization platform, especially for later (web), I will gladly take responsibility for the German translation. 👍🏻

**Comment by MickLesk** [comment]
@alextran1502 i write you an email for locali. plattform :-) (c*****.d**@g*****.com)

File: mobile/assets/i18n/de-DE.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "action_common_back": "Back",
+  "action_common_back": "Zurpck",
```
Comment (lukashass) [inline mobile/assets/i18n/de-DE.json:2]:
```suggestion
  "action_common_back": "Zurück",
```

File: mobile/assets/i18n/de-DE.json [hunk 2]
```diff
@@ -202,19 +202,19 @@
   "favorites_page_no_favorites": "Keine favorisierten Inhalte gefunden",
   "favorites_page_title": "Favoriten",
   "home_page_add_to_album_conflicts": "{added} Elemente zu {album} hinzugefügt. {failed} Elemente sind bereits vorhanden.",
-  "home_page_add_to_album_err_local": "Kann lokale Elemente noch nicht zu Alben hinzufügen, überspringe",
+  "home_page_add_to_album_err_local": "Es können lokale Elemente noch nicht zu Alben hinzugefügt werden, überspringen...",
   "home_page_add_to_album_success": "{added} Elemente zu {album} hinzugefügt.",
   "home_page_album_err_partner": "Inhalte von Partnern können derzeit nicht zu Alben hinzugefügt werden",
-  "home_page_archive_err_local": "Kann lokale Elemente nicht archvieren, überspringe",
+  "home_page_archive_err_local": "Kann lokale Elemente nicht archvieren, überspringen...",
```
Comment (lukashass) [inline mobile/assets/i18n/de-DE.json:208]:
```suggestion
  "home_page_archive_err_local": "Kann lokale Elemente nicht archivieren, überspringen...",
```

File: mobile/assets/i18n/de-DE.json (hunk 1 above)
Comment (danieldietzler) [reply mobile/assets/i18n/de-DE.json:2]: I've seen that and I'll fix it in localizely anyways, but thanks! :)

**Comment by MickLesk** [comment]
i close the pull, i transfer all into localizely :-)

**Pull request closed without merging**